app/templates/main/browse.html

```diff
@@ -0,0 +1,433 @@
+                {{ record["series"] }}
+              </a></p>
+          </dd>
+          <dd class="govuk-summary-list__value">
+            <p class="govuk-body browse__body browse__table__right-align">{{ record["last_record_transferred"] }}</p>
+          </dd>
+          <dd class="govuk-summary-list__value">
+            <p class="govuk-body browse__body browse__table__right-align">{{ record["records_held"] }}</p>
+          </dd>
+          <dd class="govuk-summary-list__value">
+            <p class="govuk-body browse__body browse__table__right-align">{{ record["consignment_in_series"] }}</p>
+          </dd>
+        </div>
+        {% endfor %}
+      </dl>
+      {% endif %}
+      {% endif %}
+      {% endif %}
+
+      {% if search_type == "transferring_body" %}
+      {% if results %}
+      {% if num_records_found > 0 %}
+      <dl class="govuk-summary-list browse-grid__list">
+        <div class="govuk-summary-list__row govuk-summary-list__row__browse-grid-list">
+          <dt class="govuk-summary-list__key browse-grid__key browse-grid__key__transferring-body">
+            Transferring body
+          </dt>
+          <dt class="govuk-summary-list__key browse-grid__key">
+            Series
+          </dt>
+          <dt class="govuk-summary-list__key browse-grid__key browse__table__right-align">
+            Last record transferred
+          </dt>
+          <dt class="govuk-summary-list__key browse-grid__key browse__table__right-align">
+            Records held
+          </dt>
+          <dd class="govuk-summary-list__key browse-grid__key browse__table__right-align">
+            Consignments within series
+          </dd>
+        </div>
+
+        <!-- Table Content -->
+        {% for record in results %}
+        <div class="govuk-summary-list__row">
+          <dd class="govuk-summary-list__value browse__table__large-width">
+            <p class="govuk-body browse__body">
+              {{ record["transferring_body"] }}
+            </p>
+            </dt>
+          <dd class="govuk-summary-list__value">
+            <p class="govuk-body browse__body">
+              <a href="{{ url_for('main.browse', series=record['series_id']) }}">
+                {{ record["series"] }}
+              </a></p>
+          </dd>
+          <dd class="govuk-summary-list__value">
+            <p class="govuk-body browse__body browse__table__right-align">{{ record["last_record_transferred"] }}</p>
+          </dd>
+          <dd class="govuk-summary-list__value">
+            <p class="govuk-body browse__body browse__table__right-align">{{ record["records_held"] }}</p>
+          </dd>
+          <dd class="govuk-summary-list__value">
+            <p class="govuk-body browse__body browse__table__right-align">{{ record["consignment_in_series"] }}</p>
+          </dd>
+        </div>
+        {% endfor %}
+      </dl>
+      {% endif %}
+      {% endif %}
+      {% endif %}
+
+
+      {% if search_type == "series" %}
```

Comment:
not going to suggest you do it now, but I think it could be worth us splitting this out into sub templates since this file is huge.

I may do that when doing the consignment browse view

Reply:
Yeah it is getting bigger with a bunch of conditional logic for each browse type. Will make a note of this.